src/libUtils/UpgradeManager.cpp

```diff
@@ -406,3 +438,55 @@ bool UpgradeManager::ReplaceNode(Mediator& mediator) {
   /// stored data
   return raise(SIGKILL) == 0;
 }
+
+bool UpgradeManager::LoadInitialDS(vector<PubKey>& initialDSCommittee) {
+  string downloadUrl =
+      "https://api.github.com/repos/KaustubhShamshery/abcdef/releases/latest";
```

Comment:
Better to avoid hardcoded URL like this. If a placeholder URL is needed, find a better alternative not bound to any specific user's repo.

Reply:
This should have been empty URL, mistake.